com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/message/Messages.java

```diff
@@ -108,14 +108,14 @@
     public static final String ERROR_DETERMINING_ACTIONS_TO_EXECUTE_ON_APP = "Error determining actions to execute on application \"{0}\"";
     public static final String ERROR_STOPPING_APP = "Error stopping application \"{0}\"";
     public static final String ERROR_STAGING_APP_1 = "Error staging application \"{0}\"";
-    public static final String ERROR_STAGING_APP_2 = "Error staging application \"{0}\": {1}";
+    public static final String ERROR_STAGING_APP_2 = "Staging of application \"{0}\" has failed. Check the application logs downloaded from the dmol command";
     public static final String ERROR_PUBLISHING_PUBLIC_PROVIDED_DEPENDENCIES = "Error publishing public provided dependecies";
     public static final String ERROR_STARTING_APP_1 = "Error starting application \"{0}\"";
-    public static final String ERROR_STARTING_APP_2 = "Error starting application \"{0}\": {1}";
+    public static final String ERROR_STARTING_APP_2 = "Application \"{0}\" startup has failed. Check the application logs downloaded from the dmol command";
     public static final String ERROR_EXECUTING_APP_1 = "Error executing application \"{0}\"";
     public static final String ERROR_EXECUTING_APP_2 = "Error executing application \"{0}\": {1}";
     public static final String ERROR_PREPARING_TO_EXECUTE_TASKS_ON_APP = "Error preparing to execute tasks on application \"{0}\"";
-    public static final String ERROR_EXECUTING_TASK_ON_APP = "Error executing task \"{0}\" on application \"{1}\"";
+    public static final String ERROR_EXECUTING_TASK_ON_APP = "Execution of task \"{0}\" has failed. Check the \"{1}\" logs downloaded from the dmol command";

```

Comment:
`"Execution of task \"{0}\" failed..."`. If "has" is removed, the message sounds like it just happened and is more concise.